pyproject.toml

```diff
@@ -0,0 +1,32 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "python_pkg_tmpl"
+version = "0.0.1"
+authors = [
+    { name="Minh Nguyen", email="minh.nguyen@uni-bremen.de" }
```

Comment:
Remove name here and put something generic

Reply:
resolved in b9df103a417ffbeb6a219563b0395f27c10a8183